android/src/main/java/com/usabilla/reactnative/ubform/UsabillaBridge.java

```diff
@@ -283,6 +262,11 @@ public Map<String, Object> getConstants() {
     public void onHostResume() {
         LocalBroadcastManager.getInstance(getReactApplicationContext()).registerReceiver(closingFormReceiver, new IntentFilter(UbConstants.INTENT_CLOSE_FORM));
         LocalBroadcastManager.getInstance(getReactApplicationContext()).registerReceiver(closingCampaignReceiver, new IntentFilter(UbConstants.INTENT_CLOSE_CAMPAIGN));
+
+        final Activity activity = getCurrentActivity();
+        if (activity instanceof FragmentActivity) {
+            usabilla.updateFragmentManager(((FragmentActivity) activity).getSupportFragmentManager());
+        }
```

Comment:
Instead of https://github.com/usabilla/usabilla-u4a-react-native/pull/169/files#r877597181, this makes sure to update it on each `onResume`.